LayoutTests/http/tests/site-isolation/accessibility/remote-element-returned.html

```diff
@@ -0,0 +1,42 @@
+<html><!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<head>
+<script src="/js-test-resources/js-test.js"></script>
+<style>
+</style>
+</head>
+<body id="body">
+
+<iframe id="iframe" onload="runTest()" src="http://localhost:8000/site-isolation/resources/iframe.html"></iframe>
+
+<script>
+var output = "This test ensures that a remote element is present in the AX tree.\n\n";
+
+function waitFor(condition) {
+    return new Promise((resolve, reject) => {
+        let interval = setInterval(() => {
+            if (condition()) {
+                clearInterval(interval);
+                resolve();
+            }
+        }, 0);
+    });
+}
```

Comment:
Is it possible for us to include `LayoutTests/resources/accessibility-helpers.js` rather than re-defining this function?

Reply:
Updated! (for http tests, `js-test-resources` is a required alias for accessing `LayoutTest/resources`)